inc/saiacl.h

```diff
@@ -227,21 +254,155 @@ typedef enum _sai_acl_table_group_attr_t
+     * @flags CREATE_ONLY
+     * @default SAI_ACL_TABLE_GROUP_TYPE_SEQUENTIAL
      */
-    SAI_ACL_TABLE_GROUP_ATTR_DUMMY = SAI_ACL_TABLE_GROUP_ATTR_START,
+    SAI_ACL_TABLE_GROUP_ATTR_TYPE,
 
     /**
      * @brief End of attributes
      */
-    SAI_ACL_TABLE_GROUP_ATTR_END
+    SAI_ACL_TABLE_GROUP_ATTR_END,
+
+    /**
+     * @brief Custom range base value start
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_CUSTOM_RANGE_START = 0x10000000,
+
+    /**
+     * @brief End of Custom range base
+     */
+    SAI_ACL_TABLE_GROUP_ATTR_CUSTOM_RANGE_END
 
 } sai_acl_table_group_attr_t;
 
 /**
+ * @brief Attribute Id for acl_table_group_member
+ */
+typedef enum _sai_acl_table_group_member_attr_t
+{
+    /**
+     * @brief Start of attributes
+     */
+    SAI_ACL_TABLE_GROUP_MEMBER_ATTR_START,
+
+    /**
+     * @brief ACL table group id
+     *
+     * By default, if the attribute is not present during table creation, SAI
+     * will internally allocate a unique group id which can be used to group
+     * tables which are created later. User would perform get operation for the
+     * already created ACL Table to fetch the group id created by SAI and use
+     * it for grouping the next table at the time of its creation.
+     * Group Id will be deleted from SAI once all the ACL Tables referencing
+     * the group are removed.
+     *
+     * In case none of the tables require grouping, SAI will assign unique group
+     * ids equal to the number of ACL Tables created and all will be independently
+     * looked up.
+     *
+     * User should always use the group id returned by SAI in the get operation
+     * to group the tables else Invalid attribute value error code will be returned.
+     *
+     * The ACL Table lookup could be done serially or in parallel. In both the
+     * cases there could be a need to group multiple tables so that only single
+     * ACL rule entry actions are performed.
+     *
+     * Within one ACL group, only one ACL entry will be hit which is based on the
+     * table priority as well as ACL entry priority within the table. Every ACL
+     * Table is associated to a priority which is a mandatory attribute needed at
+     * the time of table creation. The order of lookup within the ACL table group
+     * will be based on individual table priority. Action(s) from the ACL entry
+     * residing in the higher priority table within the group will be performed.
+     *
+     * Across the ACL groups, one hit ACL entry from each group will be selected
+     * and all non-conflicting actions from them will be performed.
+     *
+     * Without grouping, all non-conflicting actions from different ACL entry
+     * hits across the tables will be performed.
+     *
+     * @type sai_object_id_t
+     * @objects SAI_OBJECT_TYPE_ACL_TABLE_GROUP
+     * @flags CREATE_ONLY
+     * @default SAI_NULL_OBJECT_ID
+     */
+    SAI_ACL_TABLE_GROUP_MEMBER_ATTR_ACL_TABLE_GROUP_ID = SAI_ACL_TABLE_GROUP_MEMBER_ATTR_START,
```

Comment:
Why it is different from other member objects example

 @flags MANDATORY_ON_CREATE | CREATE_ONLY
     */
    SAI_NEXT_HOP_GROUP_MEMBER_ATTR_NEXT_HOP_GROUP_I

 @flags MANDATORY_ON_CREATE | CREATE_ONLY
     */
    SAI_VLAN_MEMBER_ATTR_VLAN_ID

 it should be MANDATORY_ON_CREATE since object we are creating here is group member.